winml/api/Windows.AI.MachineLearning.idl

```diff
@@ -104,21 +104,31 @@ namespace ROOT_NS.AI.MachineLearning
         //! Loads an ONNX model from a stream asynchronously.
         [remote_async]
         static Windows.Foundation.IAsyncOperation<LearningModel> LoadFromStreamAsync(Windows.Storage.Streams.IRandomAccessStreamReference modelStream);
+        //! Loads an ONNX model from a buffer asynchronously.
+        [remote_async]
```

Comment:
I think you need to add the contract version tag here as well. [contract(MachineLearningContract, 6)]. looks like this is the first one since contract 5, too, so you need to update line 36 as well.